### DISCUSSION TITLE
Other/Alternative OWASP Tools for DefectDojo

### DISCUSSION THREAD

**Discussion by berbarroso** [DefectDojo/django-DefectDojo#6259] [Q&A] [answered]
Hi!

I hope that you are all doing well,

May I ask if there are other/alternative OWASP tools that can consolidate and monitor scanned results like OWASP DefectDojo.

Thank you for the response! :D

**Comment by mtesauro** [comment, marked as answer]
I'm not aware of any OWASP tools that are similar to DefectDojo and I've been involved in OWASP since 2008 and am very familiar with OWASP projects having been on the Global Projects Committee in the past.  I'm also not aware of any open source/FLOSS tools that do the same thing as DefectDojo.

The only other tools I know of that are similar to DefectDojo are commercial tools - ThreadFix, CodeDX, Kenna, etc

HTH

**Reply by berbarroso** [reply]
Hi,

I appreciate your ideas and will try to research the tools that you recommend.

**Comment by franksec42** [comment]
you can look at appsec phoenix despite not part of owasp as project is included in your current subscription
